_automating-configurations/default-use-cases.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: default
+title: Default Use Cases
+nav_order: 10
+---
+
+# Default Use Cases
```

Comment:
I would suggest a more meaningful name for the topic. Workflow templates? Sample workflows?

Reply:
changed name to Default Workflows